x-pack/plugins/security_solution/server/plugin.ts

```diff
@@ -367,6 +372,32 @@ export class Plugin implements ISecuritySolutionPlugin {
       );
 
       plugins.data.search.registerSearchStrategy(ENDPOINT_SEARCH_STRATEGY, endpointSearchStrategy);
+
+      /**
+       * Register a config for the security guide
+       */
+      if (depsStart.cloudExperiments && i18n.getLocale() === 'en') {
```

Comment:
@afharo I've updated to only run the A/B test when the locale is english

Reply:
nice! thanks!